recipes/nanobind/conanfile.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, rename
+
+required_conan_version = ">=2.2"
```

Comment:
Is this maybe something we could try to define globally somewhere?

Reply:
> Is this maybe something we could try to define globally somewhere?

Possibly, I don't think it's too important (2.2 chosen rather arbitrarily).